packages/analytics/src/Analytics.ts

```diff
@@ -47,9 +47,11 @@ class Analytics {
   private setStoragePromise: Promise<unknown>;
 
   /**
-   * Creates a new Analytics instance with the given platform type.
-   * The analytics instance must be created by \@farfetch/blackout-react/analytics or \@farfetch/blackout-react-native/analytics.
-   * Each one of the two will properly extend this core to add all functionality needed for web or native, respectively.
+   * Creates a new Analytics instance with the given platform type. The analytics
+   * instance must be created by \@farfetch/blackout-react/analytics or
```

Comment:
```suggestion
   * instance must be created by @farfetch/blackout-react/analytics or
```

Not related to the PR scope, but is that backslash intentional? (Also happens in the line below)

Reply:
It's intentional.

The character "@" looks like a TSDoc tag, so needs to be escaped using backslash.
eslint (tsdoc/syntax) -(https://tsdoc.org/pages/packages/eslint-plugin-tsdoc)